### ISSUE TITLE
Allow user only set user data independently in PlayerUpdateBuilder/Player

### ISSUE THREAD

**Issue by zvyap** [lavalink-devs/lavalink-client#18]
In some cases, users may only want to set their user data instead of having to create the entire Track object. Lavalink only requires the user to provide an encodedTrack to be able to play music.

Creating the Track object is annoying because it requires dev.arbjerg.lavalink.protocol.v4.Track, which includes many Kotlin elements that are not user-friendly for Java developers.

Moreover, while the identifier can be modified independently, why there isn't a similar option for user data?

Edit:
What I mean for `not user-friendly`, my current solution
![image](https://github.com/lavalink-devs/lavalink-client/assets/52874570/58e4ffc7-0c71-45ab-8108-5e2f7e2fe3f1)

**Comment by duncte123** [comment]
As I replied in your other issue, I'll add a bulder for track updates.

**Comment by duncte123** [comment]
Would you mind trying version `790bc136b4972df3c1b4132de91e6d25ad8e3c66-SNAPSHOT` with the new `TrackUpdateBuilder` class? It should do what you want.

You can get this version by adding the following repository
```gradle
repositories {
    maven("https://maven.lavalink.dev/snapshots")
}
```

**Comment by zvyap** [comment]
First, idk why it got such weird thing in Java (Decompiled source not accurate)
![image](https://github.com/lavalink-devs/lavalink-client/assets/52874570/9d28b57e-cbc0-4500-83cd-440c75a46892)

But updateTrack function looks good overall, receive user data from Lavalink server not tested
![image](https://github.com/lavalink-devs/lavalink-client/assets/52874570/7960a175-cba6-4cae-9bd1-441f6ab925a2)

**Comment by duncte123** [comment]
Resolved in version `2.3.0`
